.github/workflows/lint-test.yaml

```diff
@@ -15,6 +15,8 @@ jobs:
       - name: Run chart-testing (lint)
         id: lint
         uses: helm/chart-testing-action@v1.0.0-alpha.3
+        env:
+          CT_CHECK_VERSION_INCREMENT: "false"
         with:
```

Comment:
Can we add this to release.yaml workflow as well that runs on the master branch?

Reply:
I was thinking that checking for Chart version increment in the release YAML would be a good idea, so left it.

My issue was that every PR required version change, but you only really need the version to be bumped when you release.

Happy to update this PR and add it in `release.yaml`, but think for the above reason, probably want to keep it?

Reply:
The thing is, for every commit to master, we run the `release.yaml` workflow. Which involves first running lint and test and then doing release. If there is no version increment, then release step is a no-op. But, if there is no bump, then the lint will fail on master, which is something we don't want.

Does that make sense?

Reply:
Fair enough... I'll add the `env` to release yaml